### DISCUSSION TITLE
KamaiiTV

### DISCUSSION THREAD

**Discussion by humbertovnavarro** [humbertovnavarro/KamaiiTV#21] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by supdoob** [comment]
in this for the long haul, hoping to see some promise in this platform B)

**Comment by LilyTree** [comment]
Hi. If ya'll didn't know. I like my coffee with no sugar. but peepoPog! This is looooking good.
